### PR TITLE
refactor: Add examples

### PR DESCRIPTION
**Pull request by joshua-mo-143**
## Description of change

- add axum JWT auth example
- add axum postgres example
- add actix cookie auth example
- add rocket static files example

This PR reflects https://github.com/shuttle-hq/shuttle-docs/pull/191

## How has this been tested? (if applicable)

Manual testing

### CODE REVIEW

**Comment by joshua-mo-143** [comment]
> I don't see Rocket static files in here. It should also be able to make use of Rocket.toml. See the dyn templates example.

To be honest I'm pretty tempted to just use the dynamic templating example in place of it since if you're using dynamic templating, you don't *really* need to use static file serving unless you're trying to serve an SPA

File: README.md [hunk 1]
```diff
@@ -45,30 +45,34 @@ cargo shuttle deploy
...
-This is a list of all examples maintained in this repostiory.
+This is a list of all examples maintained in this repository.
 
 **Framework** | **Name & Link** | **Description** | **Command**
 --------------|-----------------|-------------|----------------
 Actix Web | [hello-world](./actix-web/hello-world/) | Hello World | `cargo shuttle init --template actix-web`
 Actix Web | [postgres](./actix-web/postgres/) | TODO app with a Postgres DB | `cargo shuttle init --from shuttle-hq/shuttle-examples --subfolder actix-web/postgres`
+Actix Web | [static-files](./actix-web/static-files/) | Hello World page that serves static HTML and JS files | `cargo shuttle init --from shuttle-hq/shuttle-examples --subfolder actix-web/static-files`
 Actix Web | [websocket-actorless](./actix-web/websocket-actorless/) | Websocket app that checks the status of Shuttle's API | `cargo shuttle init --from shuttle-hq/shuttle-examples --subfolder actix-web/websocket-actorless`
+Actix Web | [jwt-authentication](./actix-web/cookie-authentication/) | Use JWT to authenticate API endpoints | `cargo shuttle init --from shuttle-hq/shuttle-examples --subfolder actix-web/cookie-authentication`
```
Comment (jonaro00) [inline README.md:56]:
```suggestion
Actix Web | [cookie-authentication](./actix-web/cookie-authentication/) | Use JWT to authenticate API endpoints | `cargo shuttle init --from shuttle-hq/shuttle-examples --subfolder actix-web/cookie-authentication`
```

File: rocket/static-files/src/main.rs
Comment (jonaro00) [inline rocket/static-files/src/main.rs:1]: Were we going to use Rocket.toml?

Comment (joshua-mo-143) [reply rocket/static-files/src/main.rs:1]: The static file example doesn't need Rocket.toml to work.

That being said however, we can add it if it's required

File: rocket/static-files/Cargo.toml
Comment (jonaro00) [inline rocket/static-files/Cargo.toml:1]: make sure all deps are bumped to 0.29.0 :)

**Pull request merged**